### ISSUE TITLE
Default job script template

### ISSUE THREAD

**Issue by pyotr777** [pyotr777/kportal#2]
Use default job script template when no template provided in New App form.

**Comment by pyotr777** [comment]
Now submitting an App without job template produces strange errors like this:

```
/bin/bash: -^M: invalid option
Usage: /bin/bash [GNU long option] [option] ...
/bin/bash [GNU long option] [option] script-file ...
GNU long options: --debug
--debugger
--dump-po-strings
--dump-strings
--help
--init-file
--login
--noediting
--noprofile
--norc
--posix
--protected
--rcfile
--rpm-requires
--restricted
--verbose
--version
--wordexp
Shell options: -irsD or -c command or -O shopt_option (invocation only)
-abefhkmnptuvxBCHP or -o option
```

**Comment by tuannp** [comment]
Where did you get this log ? 
from kp_server (local host) or kdeskdaemon (onto k-computer) ?

**Comment by pyotr777** [comment]
I create an App with no job template as a provider. Then I register this App as a user and submit it.
In job output files in *.sh.e* file (job stderr log) I see errors like I mentioned above.

**Comment by pyotr777** [comment]
Job files of a finished job.
![screen-shot-2016-09-20](https://cloud.githubusercontent.com/assets/485462/18658219/be23aa6c-7f3b-11e6-8417-694165349f79.png)

**Comment by pyotr777** [comment]
I wrote on github https://github.com/pyotr777/kportal/issues/2
Just to be sure reply to this mail also.

These errors are in job stderr file on K. This file is supposed to have job stderr output.  

Regards,
Peter

**Comment by pyotr777** [comment]
The above errors are in job stderr file on K. This file is supposed to have job stderr output.

**Comment by tuannp** [comment]
I just care the genterated jobscript is correct.
I think the provider user need to know how his template script work.
And you can modify the default template as what you want.
Pls try run pjsub with this jobscript. If still working maybe a bug.

**Comment by pyotr777** [comment]
Default job script template has line endings not compatible with linux, probably: \n\r instead of \n (need verification).

**Comment by pyotr777** [comment]
Fixed with 33c080f2fa13151ba279da1225ec89ed66da972a.
